### PR TITLE
More random mappings

### PR DESCRIPTION
**Pull request by magik6k**

### CODE REVIEW

File: mappings/net/minecraft/server/config/OldListsConverter.mapping [hunk 1]
```diff
@@ -0,0 +1,42 @@
+CLASS none/mq net/minecraft/server/config/OldListsConverter
```
Comment (maxpowa) [inline mappings/net/minecraft/server/config/OldListsConverter.mapping:1]: I think on a similar class for migration between versions "Upgrader" was chosen, right @copygirl?

File: mappings/net/minecraft/server/rcon/RCONBase.mapping [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		ARG 0 msg
+	METHOD a (Ljava/net/DatagramSocket;)V
+		ARG 0 socket
+	METHOD a (Ljava/net/DatagramSocket;Z)Z
+		ARG 0 socket
+	METHOD a closeSocket (Ljava/net/ServerSocket;Z)Z
+		ARG 0 socket
+	METHOD b info (Ljava/lang/String;)V
+		ARG 0 msg
+	METHOD b closeSecket (Ljava/net/ServerSocket;)Z
```
Comment (maxpowa) [inline mappings/net/minecraft/server/rcon/RCONBase.mapping:20]: Secket?

**Comment by copygirl** [comment]
I feel like you shouldn't start naming random things, especially parameters from still unnamed methods, just depending on their types. `entity`, `pos`, `damageSource`, ... doesn't really help. Might even be incorrect.

File: mappings/net/minecraft/server/config/OldListsConverter.mapping (hunk 1 above)
Comment (copygirl) [reply mappings/net/minecraft/server/config/OldListsConverter.mapping:1]: Yes, @asiekierka did that in #64 / f857ed8.

**Comment by asiekierka** [comment]
@copygirl When you rename the method, you get a chance to rename arguments. not a huge deal, though it is a bit of a waste of time yes

File: mappings/net/minecraft/server/config/OldListsConverter.mapping (hunk 1 above)
Comment (asiekierka) [reply mappings/net/minecraft/server/config/OldListsConverter.mapping:1]: ServerListUpgrader

File: mappings/net/minecraft/server/gui/ServerGui.mapping [hunk 3]
```diff
@@ -6,3 +6,4 @@ CLASS none/li net/minecraft/server/gui/ServerGui
 		ARG 0 server
 	METHOD a createWindow (Lnone/lf;)V
 		ARG 0 server
+	METHOD b playerList ()Ljavax/swing/JComponent;
```
Comment (asiekierka) [inline mappings/net/minecraft/server/gui/ServerGui.mapping:9]: getPlayerListComponent?

File: mappings/net/minecraft/server/gui/StatComponent.mapping [hunk 4]
```diff
@@ -1,4 +1,5 @@
 CLASS none/lk net/minecraft/server/gui/StatComponent
+	FIELD a FORMAT Ljava/text/DecimalFormat;
```
Comment (asiekierka) [inline mappings/net/minecraft/server/gui/StatComponent.mapping:2]: What FORMAT? Is it just a FORMAT? It's probably not just a FORMAT, I refrained from naming it before.

File: mappings/net/minecraft/server/rcon/RCONBase.mapping [hunk 5]
```diff
@@ -0,0 +1,27 @@
+CLASS none/nf net/minecraft/server/rcon/RCONBase
```
Comment (asiekierka) [inline mappings/net/minecraft/server/rcon/RCONBase.mapping:1]: RconBase, etc. not RCONBase

**Comment by asiekierka** [comment]
I don't like this PR. Too much pointless/unreliable field/method/argument naming and too many mistakes. The Rcon/MathUtils stuff is cool, you could make a separate PR with it.

**Pull request closed without merging**